### ISSUE TITLE
comment box on var nameserver

### ISSUE THREAD

**Issue by kyook** [epaule/nameserver2#26]
Please add a comment area on the variation nameserver. I would like to leave information and or references for the variations I create to make it easier for the var curator.

**Comment by epaule** [comment]
By pure luck I got Mary Ann around today, let's see what we can come up with :-)

**Comment by epaule** [comment]
I added it, but I need to get first the testing site up ... which seems to be more annoying than it supposed to be. I will give a shout when it goes live.

**Comment by epaule** [comment]
done :-)

**Comment by kyook** [comment]
Thanks!

On Fri, Jan 17, 2014 at 6:28 AM, Michael Paulini
notifications@github.comwrote:

> Closed #26 https://github.com/epaule/nameserver2/issues/26.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/epaule/nameserver2/issues/26
> .

## 

```
Karen Yook

Curator
WormBase Caltech
Tel: 626.395.5878
e-mail: kyook@caltech.edu
e-mail: kyook@wormbase.org
skype name: wbkaren
```
